File: Beginner/1019.py
x = int(input())
if x > 3600:
    horas = x//3600
    x = x%3600
else:
    horas = 0
if x > 60:
    minutos = x//60
    x = x%60
else:
    minutos = 0
segundos = x
print(f"{horas}:{minutos}:{segundos}")

File: Beginner/C/1019.c
#include <stdio.h>

int main(){

    int segundos;
    scanf("%i", &segundos);

    int horas = segundos/3600;
    segundos = segundos - (horas*3600);

    int minutos = segundos/60;
    segundos = segundos - (minutos*60);

    printf("%i:%i:%i\n",horas,minutos,segundos);

    return 0;
}